Formula/todolist.rb

```diff
@@ -0,0 +1,52 @@
+require "language/go"
+
+class Todolist < Formula
+  desc "Very fast, simple task manager for the command-line, based upon GTD."
+  homepage "http://todolist.site"
+  url "https://github.com/gammons/todolist/archive/0.2.0.tar.gz"
+  sha256 "13882bbbbcb86e05d6729f5bab2507de3aeabc1b4b4b34008480ee7549212c86"
+
+  head "https://github.com/gammons/todolist.git"
```

Comment:
Would prefer not to support `head` for Go stuff that doesn't vendor dependencies upstream unless we see massive demand to do so. We've discovered that's a world of pain all too often IRT outdated resourced dependencies.